### ISSUE TITLE
Enourmous error spamming in logs, server drive almost full...

### ISSUE THREAD

**Issue by airoons** [pterodactyl/panel#2508]
**Background (please complete the following information):**
* Panel or Daemon: Panel
* Version of Panel/Daemon: 1.0
* Server's OS: Ubuntu 18.04.5
* Your Computer's OS & Browser: Chrome

**Describe the bug**
I'll be short - about 6 hours ago I updated my panel from 0.7 to 1.0. After the update, everything seemed to work, so I left it running. Came back later with seeing errors from multiple servers of almost no "free disk space" - I panicked and found out that laravel log from pterodactyl was more than 60gb in size.

`/var/www/pterodactyl/storage/logs/laravel-2020-10-13.log` was the file. After deleting I had problems of the file being still in my servers memory, but I stopped `php artisan down`, the process eventually ended and the file got deleted...

TLDR - after updating to 1.0, the log file started spamming with these errors (and got enormously large):
https://pastebin.com/40a4gpu1

For now, I disabled laravel logs so this wouldn't happen again, but I would like to know what gone wrong...

**To Reproduce**
I am not sure, this might be a personal problem or whatnot.

**Expected behavior**
It shouldn't throw such errors as nothing was wrong when using the Panel.

**Comment by DaneEveritt** [comment]
Looks like your composer install did not go according to plan which caused critical files to be missing, which in turn caused issues with the queue worker. A bad fluke that should be fine if they've stopped now that the queue worker is restarted.
